scipy/stats/tests/test_continuous_basic.py

```diff
@@ -182,11 +212,12 @@ def test_cont_basic(distname, arg, method):
         if distname != 'truncnorm':
             check_ppf_private(distfn, arg, distname)
 
-        if distname not in skip_fit_test:
-            check_fit_args(distfn, arg, rvs[0:200], method)
+        for method in ["MLE", "MM"]:
+            if distname not in skip_fit_test[method]:
+                check_fit_args(distfn, arg, rvs[0:200], method)
 
-        if distname not in skip_fit_fix_test:
-            check_fit_args_fix(distfn, arg, rvs[0:200], method)
+            if distname not in skip_fit_fix_test[method]:
+                check_fit_args_fix(distfn, arg, rvs[0:200], method)
```

Comment:
can we make the `200` (sample size) a parameter up in the `parametrize`?

Reply:
I'm not sure why the test was written the way it was. Above, `ns=500` samples are generated for `rvs` to be used for testing moments calculations, but only 200 of them are used here. I don't see why not use all the samples for the `fit`.
I normally wouldn't parameterized either number unless I were going to run the test multiple times with different numbers, but I don't see why not to, so I went ahead.

Reply:
>I don't see why not use all the samples for the fit?

I'm not sure either, seems unlikely it was a speed thing. You could look through the history to see the commits and maybe that has some info, depending on how far back that change is it may be easy/time consuming. Not sure how helpful that would be. I agree on the parametrization piece, there is really a tradeoff here between speed to run the tests and how stable they may be with data, given this, it is ok with me to leave as is.